### PR TITLE
Web scraping academy first draft

### PR DESCRIPTION
**Pull request by mnmkng**
As agreed with @mtrunkat I'm adding the first several chapters of the Web Scraping Academy to the current docs. Once we have the `/developers` portal, we will move this part out again, to a separate instance of the "docs platform".

### CODE REVIEW

**Review by m-murasovs — CHANGES_REQUESTED** [review]
As always, excellent work! Kinda forgot that I was meant to be proof-reading and just ended up learning 😄 Also the shorter articles are a great touch - easy to navigate and not overwhelming.

I did some pedant-level edits but nothing major/structural comes to mind.

Some notes for the future:
- let's use the Oxford comma in lists: e.g. 1, 2**,** and/or 3 instead of 1, 2 and/or 3
- to highlight non-code like tab names or some text example, please make it bold and just use code highlighting for code bits
- For SEO gainz, meta descriptions should be between 120 and 150/160 characters (details vary according to source), so if you can boost those where needed, that would be cool
- It's nice to be more CTA (call to action) with links, with the link early in the sentence. So instead of "for more info, click [here](...)", it's nicer have the link at the front in CTA format: "[Learn more about functions] if you don't know functions."
- There are some varied instances of singular and plural 'data'. I don't really have a preference but it would be good to have the same one
- Emojis work quite well for me in this - maybe more emojis?

File: docs/data_collection_fundamentals/using_devtools.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    dailyPageViews: fields[3].textContent.trim(),
+    percentFromSearch: fields[4].textContent.trim(),
+    totalLinkingSites: fields[5].textContent.trim(),
+};
+console.log(result);
+```
+
+![Print collected data to the Console]({{@asset data_collection_fundamentals/images/print-website-data.webp}})
+
+If you were able to get the same result in your browser console, congratulations! You successfully collected your first data. If not, don't worry and review your code carefully. You'll crush the bug soon enough.
```
[the remark this answers is not in the document]
Comment (mnmkng) [reply docs/data_collection_fundamentals/using_devtools.md:99]: Is this good or bad? 😄

File: docs/academy.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* Advanced web scraping
+  * Advanced data collection techniques
+    * API scraping
+    * JavaScript rendering and dynamic pages
+    * Data in JavaScript objects
+  * Advanced crawling techniques
+    * Navigation with POST requests
+    * Emulating API pagination
+    * GraphQL scraping
+  * Anti*scraping protection
```
[the remark this answers is not in the document]
Comment (mnmkng) [reply docs/academy.md:42]: Yup, will fix.

File: docs/academy.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## [](#requirements) Requirements
+
+You don't need to be a developer or a software engineer to complete this course, but basic programming knowledge is recommended. Don't be afraid though. We explain everything in great detail in the Web Scraping for Beginners section and provide external references that can help you level up your web scraping and development skills. If you're new to programming, pay very close attention to the instructions and examples. A seemingly insignificant thing like using `[]` instead of `()` can make a lot of difference.
+
+As you progress to the Advanced and Pro sections, the coding will get more challenging, but still manageable to a person with an intermediate level of programming skills.
+
+## [](#next) Next up
+
+The course begins with short theory and continues with practical examples of collecting the most popular websites on the internet using your browser console. So let's get to it!
```
[the remark this answers is not in the document]
Comment (mnmkng) [reply docs/academy.md]: I was hoping the user would use the "next link" in the docs. But then I found that it's not very prominent (I thought it was a big button) and it's also under the feedback thumbs. So I'm not sure what to do now 😄

File: docs/data_collection_fundamentals/using_devtools.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+![Selecting an element with DevTools]({{@asset data_collection_fundamentals/images/selecting-first-website.webp}})
+
+Now you know where to find the name of the first website in the page's HTML, but we want to find all the website's information. To do that, hover over the elements above the one you just found, until you find the element that contains all the data about the selected website.
+
+![Selecting an element from the Elements tab]({{@asset data_collection_fundamentals/images/selecting-container-element.webp}})
+
+This element is the parent element of all the nested (child) elements and we can find it using JavaScript and collect the data. Notice the `class` attribute of the element. This is where CSS selectors become handy, because we can use them to select an element with a specific class.
+
+> Websites change and the structure or the HTML or the CSS selectors can become outdated. We'll try our best to keep this tutorial updated, but if you find that what you see on the website does not match this guide exactly, don't worry. Everything will work exactly the same. You will only have to use whatever you see on your screen and not in the screenshots here.
```
[the remark this answers is not in the document]
Comment (mnmkng) [reply docs/data_collection_fundamentals/using_devtools.md]: I don't have high hopes for this, but I can try at least 😄

File: docs/data_collection_fundamentals/devtools_continued.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+With that said, the rest of the code is just copy-pasted from the previous chapter. For each `site` in `sites`, we find all the `fields` with data for that particular `site` and save them to the `results` array by returning the object with the data. When you run the code, you should see all the data neatly structured.
+
+![Print all websites' data]({{@asset data_collection_fundamentals/images/print-all-websites-data.webp}})
+
+## [](#next) Next up
+
+And that's it. With a bit of trial and error you will be able to collect any data from a webpage that's loaded in your browser. This is a useful skill on its own. It will save you time when you need one-off data for your project. More importantly though, it taught you the basics to start programming your own scrapers. In the next chapters, we will teach you how to create your own data collection script in JavaScript and Node.js.
```
[the remark this answers is not in the document]
Comment (mnmkng) [reply docs/data_collection_fundamentals/devtools_continued.md:65]: Yeah, good catch. Will change.

File: docs/data_collection_fundamentals/using_devtools.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-murasovs) [reply docs/data_collection_fundamentals/using_devtools.md:99]: The best!

File: docs/academy.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (m-murasovs) [reply docs/academy.md]: Damn, I maintain the docs and I constantly forget the "next link" is there 😄 I'd say please link in the paragraph, just so it's obviously there for people

**Comment by mnmkng** [comment]
closing in favor of #307

**Pull request closed without merging**